index.html

```diff
@@ -378,13 +385,13 @@ <h1 style="text-align:center; padding:2em 0em 1em 0em"><strong>OUR ACHIEVEMENTS<
 				$('#anc1').click(function() {
 					$('.tab-content').css({
 						'background': 'linear-gradient( rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("images/about.jpg")  no-repeat scroll 0 0',
-						'background-size': '100%',
```

Comment:
Not required.Please refer to [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas)